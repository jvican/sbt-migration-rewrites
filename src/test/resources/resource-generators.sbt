lazy val seed = taskKey[File]("Seed.")
seed := new File(".")

lazy val seeds = taskKey[Seq[File]]("Seed.")
seeds := List(new File("."))

resourceGenerators in Compile <+= seed

(resourceGenerators in Compile) <+= seed

resourceGenerators in Compile <+= Def.task(seed.value)

(resourceGenerators in Compile) <+= Def.task {
  seed.value
}

(resourceGenerators in Compile) <+= (Def.task {
  seed.value
})

resourceGenerators in Compile <++= seeds

(resourceGenerators in Compile) <++= seeds

resourceGenerators in Compile <++= Def.task(seeds.value)

(resourceGenerators in Compile) <++= Def.task {
  seeds.value
}

(resourceGenerators in Compile) <++= (Def.task {
  seeds.value
})
