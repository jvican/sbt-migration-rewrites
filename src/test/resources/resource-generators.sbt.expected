lazy val seed = taskKey[File]("Seed.")
seed := new File(".")

lazy val seeds = taskKey[Seq[File]]("Seed.")
seeds := List(new File("."))

resourceGenerators in Compile += seed.taskValue

(resourceGenerators in Compile) += seed.taskValue

resourceGenerators in Compile += Def.task(seed.value).taskValue

(resourceGenerators in Compile) += (Def.task {
  seed.value
}).taskValue

(resourceGenerators in Compile) += (Def.task {
  seed.value
}).taskValue

resourceGenerators in Compile ++= seeds.taskValue

(resourceGenerators in Compile) ++= seeds.taskValue

resourceGenerators in Compile ++= Def.task(seeds.value).taskValue

(resourceGenerators in Compile) ++= (Def.task {
  seeds.value
}).taskValue

(resourceGenerators in Compile) ++= (Def.task {
  seeds.value
}).taskValue
